packages/common-react/src/toastify-components/stlite-kernel-with-toast.tsx

```diff
@@ -66,43 +67,55 @@ export class StliteKernelWithToast {
     };
   }
 
-  public writeFile(...args: Parameters<StliteKernel["writeFile"]>) {
-    return stliteStyledPromiseToast<void>(this.kernel.writeFile(...args), {
-      error: "Failed to write the file",
-    });
+  public writeFile(
```

Comment:
Why did you commit these changes?
They seems not related to the shared worker feature.